### PR TITLE
Add new subject `sustainability`

### PR DESCRIPTION
**Pull request by inyono**
ping @SimonKoehl 

As agreed upon in the email conversation, I added a new subject `sustainability` that should replace the current subject `permaculture`. Not sure if / what changes are necessary to the database.

### CODE REVIEW

File: src/module/Subject/config/instances.config.php [hunk 1]
```diff
@@ -142,6 +154,19 @@
                         'math-puzzle'
                     ]
                 ],
+                'sustainability' => [
+                    'allowed_taxonomies' => [
+                        'topic',
+                        'locale'
```
Comment (arekkas) [inline src/module/Subject/config/instances.config.php]: locale is missing in the german one

Comment (inyono) [inline src/module/Subject/config/instances.config.php]: fixed

**Comment by inyono** [comment]
bump @arekkas

**Comment by arekkas** [comment]
nice thanks

**Pull request merged**